Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/dao/mysql/ClassData.java

```diff
@@ -94,24 +76,13 @@ public void add(AppClass appClass) throws DAOException {
      */
     @Override
     public void update(AppClass appClass) throws DAOException {
+        
+        removeFromClass(appClass.getName(), "engine");
+        addToClass(appClass.getName(), appClass.getEngines(), "engine");
```

Comment:
I do not think this could be done with an update since we can have a different number of lines (engines per class) at the beginning and at the end of the operation (e.g. go from 1 to 2 or from 2 to 1 engines per class). 
Do you agree ?